app/Providers/KanvasAppsProvider.php

```diff
@@ -25,10 +26,10 @@ public function register(): void
     public function boot(): void
     {
         $appIdentifier = request()->header(AppEnums::KANVAS_APP_HEADER->getValue(), config('kanvas.app.id'));
-
         if (FacadesSchema::hasTable('apps') && Apps::count() > 0) {
             try {
                 $app = AppsRepository::findFirstByKey($appIdentifier);
+                (new Smtp($app))->load();
```

Comment:
@FredPeal like I mentioned before cant we just load this on the notification module? since its the only place we plan on using it for now